tools/download.sh

```diff
@@ -44,6 +44,14 @@ case "$OSTYPE" in
                         exit 2
                 esac
                 ;;
+            opensuse)
```

Comment:
LGTM but have to ask: was this verified? I don't have an openSUSE box to test on and check the contents of `/etc/os-release`.